### DISCUSSION TITLE
Error creating new SQL scirpt

### DISCUSSION THREAD

**Discussion by michaelwachalskytap** [dbeaver/dbeaver#15968] [Q&A] [answered]
When I try to create a new SQL Script from the SQL Editor menu, I get the following error:

```
Can execute command 'org.jkiss.dbeaver.core.sql.editor.create'

Reason:
A resource exists with a different case: '/General/Scripts/script.sql'.
```

    DBeaver Community Edition	22.0.1.202203181646	org.jkiss.dbeaver.ce.feature.feature.group	JKISS

Thanks for any help you can offer.

Regards,
Mike

**Comment by rajat-8** [comment, marked as answer]
Hi, 
This type of error usually comes when there is a file available already with the same name. So, in this case you either need to specify the name of the file for the new script or delete the old file with the same name.
If this isn't the case, can you provide some more information about your scenario.

regards
Rajat

**Reply by michaelwachalskytap** [reply]
Thanks Rajat - you are correct. I had created a new script file a while back and just saved it with the default name (script.sql). Never occurred to me that this would conflict with new files being created. Kind of thought it would figure that out on its own. But I've gone and renamed the offending file and now create works again. Thanks.
